### ISSUE TITLE
Is there a way to get this running inside MusicBee as a tab?

### ISSUE THREAD

**Issue by shimizurei** [milkdrop2077/MilkDrop3#31]
Can I just put the presets inside the MB plugins folder?

**Comment by milkdrop2077** [comment]
No. But you can remove the border (press f7 twice) and place the window where you want it in MusicBee. Save the position (ctrl+insert) and voila.
